sysutils/stressdrive/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+PortGroup           xcode 1.0
+
+github.setup        rentzsch stressdrive 1.3.2
+categories          sysutils
+platforms           darwin
+maintainers         @toy
```

Comment:
please also add  your e-mail address and, if possible, `openmaintainer` so that minor updates and changes can be made without your approval.

something like: `maintainers         {@toy example.org:account} openmaintainer`